<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QDialog" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student - Login System</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(223, 228, 215);</string>
       </property>
       <layout class="QVBoxLayout" name="tab">
        <item>
         <widget class="QPushButton" name="aboutMeButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>About me</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="coursesButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>Courses registered</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="logOutButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>Log out</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="aboutMePage">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="addCourseWindow">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item alignment="Qt::AlignHCenter">
                <widget class="QLabel" name="label_5">
                 <property name="styleSheet">
                  <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;</string>
                 </property>
                 <property name="text">
                  <string>My information</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>ID</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="id_label">
                     <property name="text">
                      <string>[ID]</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Name</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="name_label">
                     <property name="text">
                      <string>[Name]</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Department</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="dept_label">
                     <property name="text">
                      <string>[Department]</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Total credits</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="credits_label">
                     <property name="text">
                      <string>[TotalCredits]</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_42">
                   <item>
                    <widget class="QLabel" name="label_53">
                     <property name="text">
                      <string>CGPA</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="cgpa_label">
                     <property name="text">
                      <string>[CGPA]</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item alignment="Qt::AlignHCenter">
                   <widget class="QLabel" name="label_7">
                    <property name="styleSheet">
                     <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Update my info</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <item>
                       <widget class="QLabel" name="label_6">
                        <property name="text">
                         <string>Name</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="updateNameLineEdit"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_39">
                      <item>
                       <widget class="QLabel" name="label_8">
                        <property name="text">
                         <string>New password</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="updatePwdLineEdit"/>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="updateButton">
                  <property name="text">
                   <string>Update</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="coursesRegisterdPage">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_11">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;</string>
                </property>
                <property name="text">
                 <string>Courses registered details</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_12">
                  <item>
                   <widget class="QPushButton" name="fallButton">
                    <property name="text">
                     <string>Fall</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="springButton">
                    <property name="text">
                     <string>Spring</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="semesterLabel">
                  <property name="text">
                   <string>Semester[+]</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Preferred</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Year</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="yearComboBox"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>Select courseID registered</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="courseIdComboBox"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>Course information</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_10">
                      <item>
                       <widget class="QLabel" name="label_16">
                        <property name="text">
                         <string>Title</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="title_label">
                        <property name="text">
                         <string>[Title]</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_43">
                      <item>
                       <widget class="QLabel" name="label_18">
                        <property name="text">
                         <string>Credits</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="courseCredit_label">
                        <property name="text">
                         <string>[credits]</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_11">
                    <item>
                     <widget class="QLabel" name="label_56">
                      <property name="text">
                       <string>Students registered to the course</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QTableView" name="listOfStudents"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
